apps/api/src/app/controllers/__tests__/__snapshots__/userManagementQueries.e2e.spec.ts.snap

```diff
@@ -1207,6 +1207,72 @@ exports[`User Management Queries when the system user issuing the command is aut
       ],
       "username": "cool-james",
     },
+    {
+      "actions": [
```

Comment:
Why is this just now coming through?

Reply:
I've sorted it out. The actions are part of the view model, in this case the user view model has an actions prop. All that happened was the newly added `COSCRAD Cypress user` came through (along with available actions) in the index query response. This is expected. Note that the system user in this test is an admin and gets the actions with every query.